src/lib/Libwin/win_remote_shell.c

```diff
@@ -715,18 +739,21 @@ execute_remote_shell_command(char *remote_host, char *pipename_append, BOOL conn
 	char stdin_pipe[PIPENAME_MAX_LENGTH] = {0};
```

Comment:
Please add error login here:
https://github.com/PBSPro/pbspro/blob/bb1fa7be4315449eb94bb63e280b260c4cec3c0f/src/lib/Libwin/win_remote_shell.c#L559
https://github.com/PBSPro/pbspro/blob/bb1fa7be4315449eb94bb63e280b260c4cec3c0f/src/lib/Libwin/win_remote_shell.c#L564
mean if there is some error occurred then its not logged here